kuksa-client/setup.py

```diff
@@ -1,3 +1,15 @@
+# /********************************************************************************
```

Comment:
I thought in this files we do not need a header @SebastianSchildt ?

Reply:
I added the license as otherwise our own spdx license checker complains. As of today we require a license in all `*.py` files changed to make the checker happy. This could off course be changed, for example by explicitly excluding files called `setup.py`. But is it worth it?

Reply:
Yes, indeed. Yeah dont think it is worth it then.